plugin/plugintest/api.go

```diff
@@ -483,6 +483,7 @@ func (_m *API) GetChannelsForTeamForUser(teamId string, userId string, includeDe
 	return r0, r1
 }
 
+<<<<<<< HEAD
```

Comment:
@knrt10 Looks like merge conflicts were not fixed

Reply:
oh sorry, my bad. Updated it now @saturninoabril but I don't understand why build is failing

Reply:
I rebased with master branch